bill-nicknames.csv

```diff
@@ -32,3 +32,12 @@ hr,519,113,uafa,Uniting American Families Act of 2013
 s,296,113,uafa,Uniting American Families Act of 2013
 hr,717,113,uafa,Reuniting Families Act
 hr,83,113,cromnibus,Nickname for the Consolidated and Further Continuing Appropriations Act of 2015
+hr,5478,113,Stop Militarizing Law Enforcement Act,amends the program that allows DoD to transfer weapons and gear to
+federal and state agencies for law enforcement activities.
+hr,3361,113,USA Freedom Act, Acronym for "Uniting and Strengthening America by Fulfilling Rights and Ending
+Eavesdropping, Dragnet-collection and Online Monitoring Act" meant to halt mass data collection by the NSA and FBI.
+Failed in Nov 2014.
+hr,3361,113,End Bulk Collection Act,See "USA Freeedom Act"
+s,2685,113,The senate version of hr.3361 (USA Freedom Act / End Bulk Collection Act)
```

Comment:
Whoops; you're right!  fixing it right now and resending.

nathan

On Sun, Jan 25, 2015, at 08:21 PM, Eric Mill wrote:
> > @@ -32,3 +32,12 @@ hr,519,113,uafa,Uniting American Families Act of 2013
> >  s,296,113,uafa,Uniting American Families Act of 2013
> >  hr,717,113,uafa,Reuniting Families Act
> >  hr,83,113,cromnibus,Nickname for the Consolidated and Further Continuing Appropriations Act of 2015
> > +hr,5478,113,Stop Militarizing Law Enforcement Act,amends the program that allows DoD to transfer weapons and gear to
> > +federal and state agencies for law enforcement activities.
> > +hr,3361,113,USA Freedom Act, Acronym for "Uniting and Strengthening America by Fulfilling Rights and Ending
> > +Eavesdropping, Dragnet-collection and Online Monitoring Act" meant to halt mass data collection by the NSA and FBI.
> > +Failed in Nov 2014.
> > +hr,3361,113,End Bulk Collection Act,See "USA Freeedom Act"
> > +s,2685,113,The senate version of hr.3361 (USA Freedom Act / End Bulk Collection Act)
> 
> Looks like this line is missing the name field?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/unitedstates/bill-nicknames/pull/12/files#r23509824